game.js

```diff
@@ -1,14 +1,22 @@
 const COMP_WINS = 0;
 const USER_WINS = 1;
 
-const computerSelection = computerPlay();
-console.log(computerSelection);
+let bodyElement = document.querySelector("body");
+let resultsDiv = document.createElement("div"); //created div
+let computerChoice = document.createElement("h4"); //created h4 inside of div
+let computerSelection = computerPlay();
+computerChoice.textContent = "Computer: " + computerSelection; // put text inside h4
```

Comment:
It seems like the computer selection is displayed even before the user chooses something. Is that so? That's not a fair game :)